datadog/metric_test.go

```diff
@@ -92,27 +86,31 @@ func makeTestServer(filename string) (*fixtureHandler, *httptest.Server) {
 }
 
 func TestStackdriverDataErrors(t *testing.T) {
+	ctx := context.Background()
+
+	storage := datastore.New(ctx)
+
 	handler, server := makeTestServer("")
 	defer server.Close()
 	m, _ := NewSourceMetric("metricname", &MetricConfig{Query: "metricquery"}, time.Second, time.Hour)
 	m.client.SetBaseUrl(server.URL)
 
 	// At this point HTTP server returns 404 to all requests, so we might as well test error handling.
-	_, _, err := m.StackdriverData(testCtx, time.Now().Add(-time.Minute), &datastore.StoredMetricRecord{})
+	_, _, err := m.StackdriverData(ctx, time.Now().Add(-time.Minute), &datastore.StoredMetricRecord{Storage: storage})
 	if err == nil {
 		t.Error("expected an error when server returns 404")
 	}
 
 	// A query needs to return a single time series.
 	handler.filename = "multiple_ts.json"
-	_, _, err = m.StackdriverData(testCtx, time.Now().Add(-time.Minute), &datastore.StoredMetricRecord{})
+	_, _, err = m.StackdriverData(ctx, time.Now().Add(-time.Minute), &datastore.StoredMetricRecord{Storage: storage})
```

Comment:
Same as line 99, do this in a single.

Reply:
So genuine question here - if I do that - the function legit doesn't fit in my 27inch monitor - is it ok to leave it like this for readability sake?